pages/Product/components/Content/index.jsx

```diff
@@ -80,16 +80,16 @@ class ProductContent extends Component {
       <ProductContext.Provider value={this.state}>
         <Fragment>
           {/* IMAGE */}
-          {/* <Portal name={portals.PRODUCT_IMAGE_BEFORE} />
+          <Portal name={portals.PRODUCT_IMAGE_BEFORE} />
           <Portal name={portals.PRODUCT_IMAGE}>
-            <ImageSlider />
+            <ImageSlider productId={this.state.productId} variantId={this.state.variantId} />
```

Comment:
This feels complex. You could jut pass one id and let the component and selector take care of the rest.

Reply:
I can't. The images need to be able to fall back to the base product.